o2.sh

```diff
@@ -8,6 +8,7 @@ requires:
   - Vc
   - hijing
   - HepMC3
+  - libInfoLogger
```

Comment:
InfoLogger

Reply:
sorry, no, you are right.

Reply:
indeed, there are two packages for infologger.
libInfoLogger is the lighter one, to avoid adding the extra dependencies needed for server, DB, etc. It just has the client library instead of the full daemons set.

I have successfully used libInfoLogger in my fork of O2 (and will create the PR when this one is completed): https://github.com/sy-c/AliceO2/commits/dev